src/app/components/Board/Board.jsx

```diff
@@ -142,72 +142,74 @@ class Board extends Component {
       window.scrollTo(window.scrollX - 80, 0);
     }
   };
+
+  updateFormat = (cardId,boardLabels) => {
+    const {dispatch,cards} = this.props;
+    let newLabelFormat=[];
+    cards[cardId].labels.forEach(label => {
+      switch (label) {
+        case "inprogress":
+          boardLabels.forEach(label => {
```

Comment:
create single map object with key being: "inprogress" and value veing id. 
then just use labelIdMap[label]...